HelloPi/README.md

```diff
@@ -0,0 +1,62 @@
+
+# HelloPi
+
+A simple HelloFX application with Java 11+, JavaFX 11+ and GraalVM that runs on HotSpot or as a native image on Desktop and Embedded devices (Raspberry Pi).
+
+## Documentation
+
+Read about getting started with JavaFX on Embedded [here](https://docs.gluonhq.com/#platforms_embedded)
+
+Note that Gluon applications can run on JVM/HotSpot on ARM devices (32/64 bits) but native images can be created only for AArch64 (64 bits).
+
+## Quick Instructions
+
+We use [GluonFX plugin](https://docs.gluonhq.com/) to build a native image for platforms including desktop, android and iOS.
+Please follow the prerequisites as stated [here](https://docs.gluonhq.com/#_requirements).
+
+### Desktop
+
+Run the application on JVM/HotSpot:
```

Comment:
... on a JVM 
(adding HotSpot is a bit confusing, it can easily run on other VM's)